RatScanner/RatScannerMain.cs

```diff
@@ -33,6 +34,10 @@ public class RatScannerMain : INotifyPropertyChanged
 	private Timer _tarkovTrackerDBRefreshTimer;
 	private Timer _scanRefreshTimer;
 
+	// Check for game screen changes every 10 seconds

```

Comment:
This timer checks for changes to the game client, and triggers a new RatEye config if it does change.